packages/libraries/core/src/index.ts

```diff
@@ -200,7 +200,6 @@ export type {
 } from '@fluentui-react-native/text';
 
 export {
-  createIconProps,
```

Comment:
Can we also do a quick check downstream to make sure nobody is referring to this, in case this inadvertently breaks someone?

Reply:
That's a good point. Where are all the places I have to check to see?

Reply:
Responded offline!

Reply:
It looks like it was only used internally in FURN by other components in this repo